tests/test_io.py

```diff
@@ -959,6 +960,35 @@ def test_dict_to_hdf5_list(tmp_path: Path) -> None:
         np.testing.assert_array_equal(f["list"][()], expected["list"])
 
 
+def test_dict_to_hdf5_graincrop(dummy_graincrops_dict: grains.GrainCrop, tmp_path: Path) -> None:
+    """Test loading a GrainGrop object and writing to file."""
+    # Make a dictionary from dummy_graincrop
+    print(f"{dummy_graincrops_dict=}")
```

Comment:
```suggestion
```

Clocked I'd left this lingering in the stacked branch I'm preparing, removing it from this PR and it won't reappear.